### PR TITLE
[BH-1655] Fix memory leaks in clock faces and shortcuts

### PR DESCRIPTION
**Pull request by awulkiew**
Disable libphonenumber initialization since it is not used in Harmony but takes around 1.4MB of memory

**Your checklist for this pull request**

Make sure that this PR:
- [X] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

**Review by dawidwojtas-mudita — APPROVED** [review]

File: module-os/memory/UserMemStatsLogger.hpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    ~UserMemStatsLogger()
+    {
+        size_t fh2 = usermemGetFreeHeapSize();
+        size_t ac = usermemGetAllocationsCount();
+        size_t dc = usermemGetDeallocationsCount();
+        size_t ami = usermemGetAllocatedMin();
+        size_t ama = usermemGetAllocatedMax();
+        size_t as = usermemGetAllocatedSum();
```
Comment (SP2FET) [inline module-os/memory/UserMemStatsLogger.hpp:26]: I think that you can write more explicitly those variable names, as you did in usermem.c :P

**Review by SP2FET — COMMENTED** [review]
I'd add to the commit-msg that you added those statistics - it's not written anywhere

File: module-os/memory/UserMemStatsLogger.hpp (hunk 1 above)
Comment (awulkiew) [reply module-os/memory/UserMemStatsLogger.hpp]: done :)

**Comment by awulkiew** [comment]
@SP2FET done, thanks!

**Pull request merged**